src/main/java/org/broadinstitute/consent/http/resources/SwaggerResource.java

```diff
@@ -1,138 +1,144 @@
-                } else {
-                    response = Response.status(Response.Status.NOT_FOUND).build();
-                }
-            } else {
-                String content = FileUtils.readAllTextFromResource(swaggerResource + path);
-                if (content != null) {
-                    response = Response.ok().entity(content).type(mediaType).build();
-                } else {
-                    response = Response.status(Response.Status.NOT_FOUND).build();
-                }
-            }
+          response = Response.status(Response.Status.NOT_FOUND).build();
         }
-        return response;
-    }
-
-    private String getMediaTypeFromPath(String path) {
-        String mediaType;
-        switch (StringUtils.substringAfterLast(path, ".")) {
-            case "css":
-                mediaType = MEDIA_TYPE_CSS;
-                break;
-            case "js":
-                mediaType = MEDIA_TYPE_JS;
-                break;
-            case "png":
-                mediaType = MEDIA_TYPE_PNG;
-                break;
-            case "gif":
-                mediaType = MEDIA_TYPE_GIF;
-                break;
-            default:
-                mediaType = MediaType.TEXT_HTML;
-                break;
+      } else {
+        String content = FileUtils.readAllTextFromResource(swaggerResource + path);
+        if (content != null) {
+          response = Response.ok().entity(content).type(mediaType).build();
+        } else {
+          response = Response.status(Response.Status.NOT_FOUND).build();
         }
-        return mediaType;
+      }
     }
+    return response;
+  }
 
-    private String getIndex(String swaggerResource) {
-        String content = FileUtils.readAllTextFromResource(swaggerResource + "index.html");
-        return content
-            .replace("your-client-id", config.getClientId())
-            .replace("your-client-secret-if-required", "")
-            .replace("your-realms", "Broad Institute")
-            .replace("your-app-name", "Consent")
-            .replace("scopeSeparator: \",\"", "scopeSeparator: \" \"")
-            .replace("jsonEditor: false,", "jsonEditor: false," + "validatorUrl: null, apisSorter: \"alpha\", operationsSorter: \"alpha\",")
-            .replace("url = \"http://petstore.swagger.io/v2/swagger.json\";", "url = '/api-docs/api-docs.yaml';");
+  private String getMediaTypeFromPath(String path) {
+    String mediaType;
+    switch (StringUtils.substringAfterLast(path, ".")) {
+      case "css":
+        mediaType = MEDIA_TYPE_CSS;
+        break;
+      case "js":
+        mediaType = MEDIA_TYPE_JS;
+        break;
+      case "png":
+        mediaType = MEDIA_TYPE_PNG;
+        break;
+      case "gif":
+        mediaType = MEDIA_TYPE_GIF;
+        break;
+      default:
+        mediaType = MediaType.TEXT_HTML;
+        break;
     }
+    return mediaType;
+  }
 
+  private String getIndex(String swaggerResource) {
```

Comment:
Are the \n line breaks necessary here?

Reply:
Technically, no, they're not required. It was helpful in debugging the rendered javascript though. Getting the configs syntactically correct was a bit of a challenge.